hazelcast/src/main/java/com/hazelcast/config/WanReplicationRef.java

```diff
@@ -132,6 +141,15 @@ public WanReplicationRef setRepublishingEnabled(boolean republishEnabled) {
         return this;
     }
 
+    public boolean isPersist() {
```

Comment:
Can you add some javadoc? I know the rest of the class is heavily missing it but we can improve it :)